data/maps/zone1/act1/entities/interactables.csv

```diff
@@ -0,0 +1,233 @@
+Adv2,INTERACTABLES,78,12,,,,,
+34,0,24,12,TOGGLE__CHECKPOINT,0,0,0,0
```

Comment:
If these values represent the region and position, does it not make sense to just store the absolute position and use the tool to convert to these values automatically? What does @MainMemory think?

Reply:
And for her opinion, I am more looking for if we need to think about if these map editors need to contain the logic to know that entity positions are not stored as absolute values? Wouldn't it be easier for people to write tools if this format just stored the absolute position

Reply:
I did that before (that was kind of the "initial draft" of the format) but had some complications with that.

This way it's MUCH more straightforward to convert in either direction.

Reply:
Yes of course, but then doesn't the editor have to know how to generate these values?

Reply:
Yeah, but I don't think that's a big deal.

Reply:
I mean, the Tiled addons already have that code, and so would a potential dedicated editor I might make, whether or not I adopt this format for my tools.